include/libsupl.rst

```diff
@@ -0,0 +1,153 @@
+.. _libsupl:
+
+LibSUPL
+#######
+
+LibSUPL is an external SUPL (Secure User Plane Location) client library, which is used to retrieve `A-GPS`_ data from a SUPL server using the SUPL protocol.
+
+.. |lib_note| replace:: Therefore, you must download and install it separately.
+
+.. include:: supl_os_client.rst
+   :start-after: supl_details_start
+   :end-before: supl_details_end
+
+See :ref:`download_supl` for information on downloading the external SUPL client library.
+
+Overview
+********
+SUPL is an Enabler, which enables the transfer of assistance and network based positioning data of a SET (SUPL Enabled Terminal) over an IP User Plane bearer.
+See `SUPL context model`_ for more information on the SUPL architecture.
+SUPL is also designed to support future positioning technologies.
+
+SUPL uses A-GPS for obtaining GPS data, overcoming the limitations of standalone GPS.
+
+A standalone GPS uses the orbital data (like `Ephermeridus`_ and the `Almanac`_) of the satellite to calculate the current position of a device.
+Typically, the data download from the satellite takes a long time and the first position is calculated in 30-40 seconds.
+Any loss of signal requires the whole process to be restarted.
+
+An A-GPS server is a cached server that stores the orbital data.
+This information can be downloaded at a fast rate since an A-GPS capable device supports mobile radio-network bearers with high data rate such as GSM or LTE.
+
+The SUPL version supported by this library is 2.0.4.
+
+LibSUPL payload
+***************
+
+`LPP (LTE Positioning Protocol)`_ is the supported SUPL payload format.
+
+LPP is a point-to-point protocol used between a location server (for example, SLP) and a target device (for example, SET) to position the device using position-based information obtained from reference sources.
+A SET or SUPL Enabled Terminal is a logical part of a device that supports communication with a SUPL network.
```

Comment:
SET has already been introduced as abbreviation, so no need to give both versions here.